name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/santanderconsumerbank/Kontoauszug01.txt

```diff
@@ -8,12 +8,13 @@ Kontoauszug Erstellungsdatum: 03.06.2023
 Kontoinhaber: Max Muster
 BestFlex Eröffnungsdatum: 11.05.2023
 Kontostand per 11.05.2023 Kontostand per 31.05.2023 Summe Gutschriften Summe Belastungen Zinserträge Steuern (KeSt bzw. QueSt)
-€ 0,00 € 38,98 € 40,64 € 1,66 € 6,63 € 1,66
+€ 0,00 € 37,98 € 40,64 € 2,66 € 6,63 € 1,66
 Buchungsdatum Valutadatum Betrag Saldo Buchungstext
 15.05.2023 15.05.2023 0,01 0,01 Einzahlung von Konto  AT123456789012345678 lautend auf Max Muster
 19.05.2023 19.05.2023 34,00 34,01 Einzahlung von Konto  AT123456789012345678 lautend auf Max Muster
-31.05.2023 31.05.2023 6,63 40,64 Zinsgutschrift Habenzinsen
-31.05.2023 31.05.2023 -1,66 38,98 Kapitalertragsteuer
+20.05.2023 20.05.2023 -1,00 33,01 Auszahlung auf Konto IBAN AT123456789012345678
+31.05.2023 31.05.2023 6,63 39,64 Zinsgutschrift Habenzinsen
+31.05.2023 31.05.2023 -1,66 37,98 Kapitalertragsteuer
```

Comment:
_"The test files should not be modified beyond the anonymization."_
- see [Contributing rules --> Test cases](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#test-cases-1) .... Please note in future.